### DISCUSSION TITLE
Saved Views

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#512] [Ideas]
There are times, sometimes once a month, that I need to open one of my mindmaps and have it show a very specific view - certain parents and children nodes, generally that fit on a single screen. To set up these views every time can be quite time-consuming. 

I would find it very helpful to be able to open up a specific set of nodes, and then instruct FP to _save this view_. Ideally, I could give the view a unique name, so that the view could be called by its name, and so that I could have multiple saved views.

If this is not clear, please ask questions. Thanks.

~ Bal

**Comment by EdoFro** [comment]
Are you talking about remembering which nodes are folded or about zoom and position of the map on the window?

**Reply by wordmuse** [reply]
Thanks for asking - that's helpful for clarifying what I want.

I want to preserve which notes are folded and not folded. And now that you mention zoom - if the zoom state could also be remembered, that would be super.

~ Bal

**Comment by dpolivaev** [comment]
I think it is exactly what presentations do.

**Reply by wordmuse** [reply]
I will check them out. The word presentations made me think it was read only. But if I can continue creating, editing and deleting nodes while viewing a presentation, that very well might be what I need. Thank you.

**Reply by euu2021** [reply]
To be honest, today was the first time that  I opened the Presentations features, and it's pretty interesting. Fp always surprises me.

**Comment by wordmuse** [comment]
I've started exploring Presentations and find it a bit confusing. Are there any documentation or videos about creating, editing, using and deleting presentations?

Thanks.
~ Bal

**Reply by macmarrum** [reply]
It can be confusing at first.

I find it helpful to treat a `Slide` as a `Saved view`, and a `Presentation` as a group of `Saved views`.

Here's a simple tutorial

1. Create a new `Presentation` and give it a name or leave the default _(New presentation 1)_
2. Create a new `Slide` and give it a name or leave the default _(New slide 1)_
3. In your mindmap, select the nodes which should be visible when the view is applied
4. In `Presentations->Slide content` top row, click the left-most button `Make currently selected nodes the slide content`
5. Make sure `Show only selected nodes` checkbox is checked
6. Apply the view by running the presentation (the play button at the bottom)

If you add more then one `Slide` to a `Presentation`, you can switch between them in a sequence, using the play controls.

**Reply by euu2021** [reply]
So, when you want to open multiple map views on certain places each, you go like:

1. Start presentation
2. Slide 1
3. new map view
4. go to Slide 2
5. New map view
6. go to Slide 3
7. ...

**Reply by wordmuse** [reply]
Thank you! Looks simple enough. I'll give it a try today.

~ Bal
